conf/build.manifest

```diff
@@ -294,6 +299,8 @@
 				"pkg-pybonjour",
 				"pydevd",
 				"python-netsnmpagent",
+				"python-pyroute2",
+				"python-remote-pdb",
```

Comment:
Do we need these two here? isnt in middlewared repo enough?

Reply:
Looks like we do, this package wasn't built during the last build

Reply:
So yes for python-pyroute2, but no for python-remote-pdb ?

Reply:
How is it different? The same thing will happen if we upgrade it, it will only be rebuild during full build.

Reply:
I missed the remote-pdb being a repo of our own